app/static/src/App.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div id="app" class="container">
+    <h1 class="text-center mt-3">Shopcart Service</h1>
```

Comment:
we should just name it "Shopcarts REST API Service" so that the Behave test pass. right?

Reply:
We can change it or change the test. I'm not by a computer right now so if someone else wants to change it and rebuild via `npm run build`. Or we can merge and change it in a different PR.